services/ui-src/src/forms/mcpar/ebssei/ebssei.json

```
@@ -1,18 +1,159 @@
 {
   "path": "/mcpar/bss-entity-indicators",
   "intro": {
-    "section": "E: BSS Entity Indicators",
-    "subsection": "X: Program Integrity"
+    "section": "Section E: BSS Entity Indicators",
+    "subsection": "Topic IX. Beneficiary Support System (BSS) Entities",
+    "spreadsheet": {
+      "title": "Find in the Excel Workbook",
+      "descriptionList": ["E_BSS_Entities"]
+    },
+    "info": [
+      {
+        "type": "html",
+        "content": "Per 42 CFR 438.66(e)(2)(ix), the Managed Care Program Annual Report must provide information on and an assessment of the operation of the managed care program including activities and performance of the beneficiary support system. Information on how BSS entities support program-level functions is on the <a href=\"/mcpar/program-information/add-bss-entities\">Program-Level BSS</a> page."
+      }
+    ]
```

This is the exact type of scenario that we set up html parsing for. The best way to do this and keep the link running through react router is using the `internalLink` type, like this:
```suggestion
    "info": [
      {
        "type": "html",
        "content": "Per 42 CFR 438.66(e)(2)(ix), the Managed Care Program Annual Report must provide information on and an assessment of the operation of the managed care program including activities and performance of the beneficiary support system. Information on how BSS entities support program-level functions is on the "
      },
      {
        "type": "internalLink",
        "content": "Program-Level BSS",
        "props": {
          "to": "/mcpar/program-information/add-bss-entities"
        }
      },
      {
        "type": "span",
        "content": " page."
      }
    ]
```